ambari-project/pom.xml

```diff
@@ -206,6 +206,12 @@
         <artifactId>mockito-core</artifactId>
         <version>1.10.19</version>
       </dependency>
+      <dependency>
+        <groupId>org.hamcrest</groupId>
```

Comment:
I think that this BSD license is compatible. Just wanted to mention it for a double-check.

Reply:
Yes it's BSD, also used in logsearch.